.env

```diff
@@ -0,0 +1,9 @@
+MYSQLDB_USER=root
```

Comment:
It's better to create .env.sample file and don't hardcode properties
other devs should set up their own properties